### PR TITLE
[GH-1306] Add `workflows` multimethod for workloads

### PR DESCRIPTION
**Pull request by ehigham**
RR: https://broadinstitute.atlassian.net/browse/GH-1306
Add workflows multimethod
Change all locations where workflows are accessed via the `:workflows` keyword to use this function.
Implement `workflows` for COVID
- I've added a simple test to make sure the SQL is correct
- Awaiting executor implementation to test with more than 0 workflows.

### CODE REVIEW

File: api/src/wfl/module/covid.clj
Comment (okotsopoulos) [inline api/src/wfl/module/covid.clj:368]: Awesome, I look forward to using this for [GH-1315](https://broadinstitute.atlassian.net/browse/GH-1315).

File: api/src/wfl/module/batch.clj
Comment (okotsopoulos) [inline api/src/wfl/module/batch.clj:97]: What is this function's eventual fate, guessing it's here for backwards compatibility?  Is there a known condition that would allow us to delete it?

File: api/src/wfl/module/covid.clj
Comment (okotsopoulos) [inline api/src/wfl/module/covid.clj:258]: Thanks.

File: api/test/wfl/integration/modules/covid_test.clj
Comment (okotsopoulos) [inline api/test/wfl/integration/modules/covid_test.clj:121]: Thanks for adding this test as well.  Nit: might be good to break out this assignment into a helper since we use it in the above test.

File: api/src/wfl/module/batch.clj
Comment (ehigham) [reply api/src/wfl/module/batch.clj:97]: yes I think there is one.
In the future I propose that we add a "InputsListSource" - a source that's literally a list of workflow inputs:
```json
...
"source": {
    "name": "Inputs List",
    "inputs": [{
        "sample_name": "Sample",
        ...
    },
    ...
    ]}},
...
```

This would allow us to "batch" workflows in the way we currently do but recast into this source->executor->sink model.

**Review by okotsopoulos — APPROVED** [review]
LGTM, this would help me to merge but you may like to wait for the approval of someone with more background knowledge on the historical code being altered.

File: api/test/wfl/integration/modules/covid_test.clj
Comment (ehigham) [reply api/test/wfl/integration/modules/covid_test.clj]: Sure. Nit: it's not an assignment, it's a binding :-p

**Pull request merged**
